library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -60,15 +63,23 @@ public EnvoyConfiguration(String statsDomain, int connectTimeoutSeconds, int dns
    * @throws ConfigurationException, when the template provided is not fully
    *                                 resolved.
    */
-  String resolveTemplate(final String templateYAML, final String filterTemplateYAML) {
+  String resolveTemplate(final String templateYAML, final String platformFilterTemplateYAML, final String nativeplatformFilterTemplateYAML) {
```

Comment:
`nativeplatformFilterTemplateYAML` shouldn't this be `nativeFilterTemplateYAML`? Also should be documented above because I don't think this is immediately obvious as to what should be passed here (it's a bit of a "magic" string)

Reply:
Can you actually elaborate on how this will be used directly? I.e., what will a consumer need to specify to make this work?

Reply:
bad find and replace. will fix.